tasking/pulp/tasking/scheduler.py

```diff
@@ -258,7 +258,7 @@ def tick(self):
                           % {'celerybeat_name': self.celerybeat_name})
             ret = self.call_tick(self, self.celerybeat_name)
         except TaskLock.DoesNotExist:
-            TaskLock.objects.get(name=self.celerybeat_name, timestamp__lte=old_timestamp).delete()
+            TaskLock.objects.filter(name=self.celerybeat_name, timestamp__lte=old_timestamp).delete()
```

Comment:
I think this line isn't needed at all. `TaskLock` is unique by `name` and because of the query at L#253 causing a `TaskLock.DoesNotExist` exception, `TaskLock.objects.filter(name=self.celerybeat_name).count()` should equal 0.